lib/ui/semantics.dart

```diff
@@ -348,6 +349,14 @@ class SemanticsFlag {
   ///   * [SemanticsFlag.hasCheckedState], which enables a checked state.
   static const SemanticsFlag isChecked = SemanticsFlag._(_kIsCheckedIndex);
 
+  /// Whether a tristate checkbox is in its mixed state.
+  ///
+  /// Should only be true when the checkbox has [tristate] = true.
+  ///
+  /// Should be false when the checkbox is either checked or unchecked.
```

Comment:
did you mean `Should be true`?

Reply:
No, why would that be the case? If the checkbox is either fully checked or fully unchecked, the mixed state should be false.

Reply:
oh my bad, I misread the sentence